crates/pallet-executor/src/lib.rs

```diff
@@ -294,11 +327,18 @@ mod pallet {
     }
 
     #[repr(u8)]
-    enum InvalidTransactionCode {
+    pub enum InvalidTransactionCode {
         BundleEquivicationProof = 101,
```

Comment:
The intention is to let the error code of each pallet have its own scope of errors, otherwise, we can't precisely tell the error with just the custom error code if multiple pallets define the error from `0`, some substrate pallets already use `0`.

Reply:
Hm, if we just have `0..=255` available here, start at `101` feels like an arbitrary choice. In which context we might have this ambiguity of error codes, isn't it transaction-specific?

Reply:
Yes, it's an arbitrary choice at present, we can redesign this to make it more formal when the codebase becomes more stable. All invalid transactions using custom error collapse to this global `Custom(u8)` type, it's not pallet or transaction-specfic.